static.webpack.config.common.js

```diff
@@ -33,12 +32,11 @@ class MergePortalConfigsPlugin {
         const output = {}
         const portalDirectory = 'portals'
         const directories = fs.readdirSync(portalDirectory, { withFileTypes: true })
-          .filter((dirent) => dirent.isDirectory())
+          .filter((dirent) => dirent.isDirectory() && !dirent.name.startsWith('.'))
```

Comment:
I think this whole file is going to be changed regardless since we are going to be using the index.js and hardcoding the names in the other approach